tests/kafkatest/tests/streams/streams_smoke_test.py

```diff
@@ -31,48 +31,80 @@ def __init__(self, test_context):
             'echo' : { 'partitions': 5, 'replication-factor': 1 },
             'data' : { 'partitions': 5, 'replication-factor': 1 },
             'min' : { 'partitions': 5, 'replication-factor': 1 },
+            'min-suppressed' : { 'partitions': 5, 'replication-factor': 1 },
+            'min-raw' : { 'partitions': 5, 'replication-factor': 1 },
             'max' : { 'partitions': 5, 'replication-factor': 1 },
             'sum' : { 'partitions': 5, 'replication-factor': 1 },
+            'sws-raw' : { 'partitions': 5, 'replication-factor': 1 },
+            'sws-suppressed' : { 'partitions': 5, 'replication-factor': 1 },
             'dif' : { 'partitions': 5, 'replication-factor': 1 },
             'cnt' : { 'partitions': 5, 'replication-factor': 1 },
             'avg' : { 'partitions': 5, 'replication-factor': 1 },
             'wcnt' : { 'partitions': 5, 'replication-factor': 1 },
             'tagg' : { 'partitions': 5, 'replication-factor': 1 }
         })
 
+        self.test_context = test_context
         self.driver = StreamsSmokeTestDriverService(test_context, self.kafka)
-        self.processor1 = StreamsSmokeTestJobRunnerService(test_context, self.kafka)
-        self.processor2 = StreamsSmokeTestJobRunnerService(test_context, self.kafka)
-        self.processor3 = StreamsSmokeTestJobRunnerService(test_context, self.kafka)
-        self.processor4 = StreamsSmokeTestJobRunnerService(test_context, self.kafka)
 
-    @cluster(num_nodes=9)
-    def test_streams(self):
-        """
-        Start a few smoke test clients, then repeat start a new one, stop (cleanly) running one a few times.
-        Ensure that all results (stats on values computed by Kafka Streams) are correct.
-        """
+    @cluster(num_nodes=8)
+    @matrix(eos=[True, False], crash=[True, False])
+    def test_streams(self, eos, crash):
+        #
+        if eos:
+            processor1 = StreamsSmokeTestEOSJobRunnerService(self.test_context, self.kafka)
+            processor2 = StreamsSmokeTestEOSJobRunnerService(self.test_context, self.kafka)
+            processor3 = StreamsSmokeTestEOSJobRunnerService(self.test_context, self.kafka)
+        else:
+            processor1 = StreamsSmokeTestJobRunnerService(self.test_context, self.kafka)
+            processor2 = StreamsSmokeTestJobRunnerService(self.test_context, self.kafka)
+            processor3 = StreamsSmokeTestJobRunnerService(self.test_context, self.kafka)
```

Comment:
I went down to 3 processors and changed up the test scenario a little.

Instead of starting two nodes concurrently at the start, we just start one so that we can be sure it will process some data before we kill it. The same logic applies to the second round of processing and kill.